Sources/Sentry/SentryDateUtil.m

```diff
@@ -38,6 +38,11 @@ + (NSDate *_Nullable)getMaximumDate:(NSDate *_Nullable)first andOther:(NSDate *_
     }
 }
 
++ (long)javascriptDate:(NSDate *)date
```

Comment:
we're taking a Date and returning an long representing that date in milliseconds, right?
How about:

```suggestion
+ (long)toMillisecond:(NSDate *)date
```